src/Writers/HTMLWriter.jl

```diff
@@ -901,6 +910,15 @@ analytics_script(tracking_id::AbstractString) =
         """
     )
 
+function warning_script(should_warn)
+    if should_warn
+        return Tag(:script)[Symbol(OUTDATED_VERSION_ATTR)](
+            read(joinpath(ASSETS, "warner.js"), String)
+        )
+    end
+    return Tag(Symbol("#RAW#"))("")
+end
+
```

Comment:
Hmm. What pros and cons do you see between (1) pasting into the HTML, (2) loading it from a file via a `<script>` tag (like we do for `themeswap.js`), or (3) loading via RequireJS like we do for most deps?

I guess RequireJS adds a small lag, which means the page renders at first without the box and then it appears, which can be annoying? But I feel that loading it from a separate file would be cleaner and consistent with `themeswap.js`? Loading time shouldn't be an issue with a separate file, since it seems to be working fine for the theme.

Reply:
Makes sense, I'll switch this to directly loading from the file.
The reason I stopped using RequireJS is to get a tag with `OUTDATED_VERSION_ATTR` into the source code so the accompanying PR in DocumenterTools can check whether it should add a warning to older pages.